blocks/api/parser.js

```diff
@@ -19,6 +19,22 @@ import { isValidBlock } from './validation';
 import { getCommentDelimitedContent } from './serializer';
 import { attr, prop, html, text, query, node, children } from './matchers';
 
+/**
+ * Higher-order hpq matcher which enhances an attribute matcher to return true
+ * or false depending on whether the original matcher returns undefined. This
+ * is useful for boolean attributes (e.g. disabled) whose attribute values may
+ * be technically falsey (empty string), though their mere presence should be
+ * enough to infer as a truthy value.
+ *
+ * @param {Function} matcher Original hpq matcher.
+ *
+ * @return {Function} Enhanced hpq matcher.``
```

Comment:
Not sure what the backticks at the end are for 🤷‍♂️

Reply:
>Not sure what the backticks at the end are for 🤷‍♂️

Fat fingered 😄

Reply:
That was my guess 😆